src/dmd/backend/cod1.d

```diff
@@ -3315,7 +3315,7 @@ void cdfunc(ref CodeBuilder cdb, elem* e, regm_t* pretregs)
                     regm_t tosave = keepmsk & retregs;
 
                     // tosave is the mask to save and restore
-                    for (int j = 0; tosave; j++)
+                    for (reg_t j = 0; tosave; j++)
```

Comment:
???

Reply:
The subsequent code uses `j` as a register number.

Reply:
Fair enough.